src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/config/OpenApiConfiguration.kt

```diff
@@ -41,24 +47,27 @@ class OpenApiConfiguration(
     )
     .components(
       Components().addSecuritySchemes(
-        "bearer-jwt",
+        SCHEME_HDC_ADMIN,
```

Comment:
This change was to fix [broken swagger docs](https://github.com/ministryofjustice/hmpps-prisoner-search/pull/155)